factory/core/coreComponents.go

```diff
@@ -224,6 +224,7 @@ func (ccf *coreComponentsFactory) Create() (*coreComponents, error) {
 	}
 
 	alarmScheduler := alarm.NewAlarmScheduler()
+	// TODO: disable watchdog if block processing cutoff is enabled
```

Comment:
implement TODO?
I do not think the watchdog is disabled. 
The exceptions for the watchdog are implemented in this function
https://github.com/multiversx/mx-chain-go/blob/4a8523eee2871e2e4202c6e278d1cca91e9ff292/factory/consensus/consensusComponents.go#L310

Reply:
If block processing cutoff is enabled, then I enable the flag that disables the watchdog (aka I disable the watchdog)